src/Forms/ReviewContentHandler.php

```diff
@@ -120,12 +120,19 @@ public function submitReview($record, $data)
      */
     public function canSubmitReview($record)
     {
-        if (!$record->canEdit()
-            || !$record->hasMethod('canBeReviewedBy')
-            || !$record->canBeReviewedBy(Security::getCurrentUser())
-        ) {
+        // Ensure the parameter of correct data type
+        if (!$record instanceof DataObject) {
             return false;
         }
-        return true;
+
+        $user = Security::getCurrentUser();
+        // Whether or not the user is a reviewer. User must be allowed to view the
+        $isReviewer = $record->canView($user) &&
```

Comment:
`$isReviewer` + `$canEdit` is code-duplication with the new `canUseReviewContent()` method above in this PR.  Is there a way that we can reduce this to a single-occurance?

Reply:
Can create a trait that contains the similar lines of code and re-use that in these 2 classes, would that be ok?

Reply:
Sure, trait, extension, public static method, service class, use whatever makes the most sense given the rest of the code in the module